.github/workflows/main.yaml

```diff
@@ -10,6 +10,7 @@ on:
 jobs:
   lint:
     runs-on: ubuntu-latest
+    timeout-minutes: 30
```

Comment:
Why not use the default value, can you pls explain a little bit here or in the commit message on why we adopt "30" mins?

Reply:
Well, will add the detail to PR description :) Because the default timeout value is 360 mins, it's too long to report the result if the job get stucked. And usually our jobs will not run for more than 20 mins.

Reply:
thanks!